src/components/Navbar.js

```diff
@@ -1,41 +1,35 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import '../App.css';
 
-const Navbar = () => {
-  const links = [
-    {
-      id: 1,
-      path: '/books',
-      text: 'Books',
-    },
-    {
-      id: 2,
-      path: '/categories',
-      text: 'Categories',
-    },
-  ];
-  return (
-    <header>
-      <nav>
-        <div className="menu">
-          <h1>Bookstore CMS</h1>
-          <ul>
-            {links.map((link) => (
-              <li key={link.id}>
-                <Link
-                  to={link.path}
-                  className="nav-link"
-                  exact="true"
-                >
-                  {link.text}
-                </Link>
-              </li>
-            ))}
-          </ul>
-        </div>
-      </nav>
-    </header>
-  );
-};
+const Navbar = () => (
+  <nav className="navbar navbar-expand-lg navbar-light bg-light m-4">
+    <div className="container-fluid">
+      <a className="navbar-brand  fw-bold" href="/">
+        <Link className="nav-link" to="/">
+          <h4 className="rangi"> Bookstore CMS </h4>
+        </Link>
+      </a>
```

Comment:
- [ ] I think you don't need to wrap a `Link` element in 'a` tag as they are the same when converted to html. The browser is complaining about it. 

![Screenshot from 2021-10-08 12-36-01](https://user-images.githubusercontent.com/64894927/136533919-f80dfa94-55d9-4ec6-a40f-9a2987d477c0.png)

Reply:
Thank you for letting me know. I changed what you requested me to do.